### DISCUSSION TITLE
Data modification error

### DISCUSSION THREAD

**Discussion by iPOiZN** [visgl/deck.gl#8701] [Q&A]
`deck: matching of PolygonLayer({id: 'h3Hexagon-layer-hexagon-cell-hifi'}): 'get' on proxy: property '__diff' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value (expected '[object Array]' but got '[object Array]') TypeError: 'get' on proxy: property '__diff' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value (expected '[object Array]' but got '[object Array]')`

I am getting this error when changing data. Initially the data is loaded as shown on the site -  

```
// Create an async iterable
async function* getData() {
  for (let i = 0; i < 10; i++) {
    await const chunk = fetchChunk(...);
    yield chunk;
  }
}

function render() {
  const layer = new ScatterplotLayer({
    // When a new chunk arrives, deck.gl only updates the sub buffers for the new rows
    data: getData(),
    ...
  });

  deck.setProps({layers: [layer]});
}
```

Please help to resolve this.

**Comment by Pessimistress** [comment]
If you are using Vue, https://vuejs.org/api/reactivity-advanced.html#markraw

> Some values simply should not be made reactive, for example a complex 3rd party class instance, or a Vue component object.
